docs/CACHING.md

```diff
@@ -0,0 +1,353 @@
+```
+
+With this configuration, once your workload is scheduled into a node, Mountpoint CSI Driver will schedule a Mountpoint Pod to the same node with the `ephemeral` volume. EBS CSI Driver will then dynamically provision an EBS volume and attach it to the node for Mountpoint to use as cache.
+
+The EBS volume and the Mountpoint Pod (therefore it's ephemeral PVC) will automatically cleaned up once the workload is terminated. We highly recommend you to use `reclaimPolicy: Delete` in your StorageClass to ensure the cache PV is automatically cleaned up as part of this process.
+
+##### Using Local Volume Static Provisioner to use local NVMe
+
+Some Amazon EC2 instances offer non-volatile memory express (NVMe) solid state drives (SSD) instance store volumes. You can utilise [Local Volume Static Provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner) to use instance store as cache. See [Instance store volume limits for EC2 instances](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-store-volumes.html) for more details about instance store support on EC2 instances, and [EKS Persistent Volumes for Instance Store](https://aws.amazon.com/blogs/containers/eks-persistent-volumes-for-instance-store/) on using instance storage in EKS.
+
+The Local Volume Static Provisioner allows you to configure some options, you can find more details in their [Getting started guide](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/getting-started.md).
+
+As an example, you can configure your [eksctl](https://eksctl.io/) configuration to mount available NVMe instance storage disks at `/dev/disk/kubernetes`:
+
+```yaml
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+metadata:
+  name: cluster-with-storage
+  region: eu-central-1
+managedNodeGroups:
+  - name: storage-nvme
+    desiredCapacity: 1
+    instanceType: i3.8xlarge
+    amiFamily: AmazonLinux2023
+    preBootstrapCommands:
+      - |
+          cat <<EOF > /etc/udev/rules.d/90-kubernetes-discovery.rules
+          # Discover Instance Storage disks so kubernetes local provisioner can pick them up from /dev/disk/kubernetes
+          KERNEL=="nvme[0-9]*n[0-9]*", ENV{DEVTYPE}=="disk", ATTRS{model}=="Amazon EC2 NVMe Instance Storage", ATTRS{serial}=="?*", SYMLINK+="disk/kubernetes/nvme-\\\$attr{model}_\\\$attr{serial}", OPTIONS="string_escape=replace"
+          EOF
+      - udevadm control --reload && udevadm trigger
+```
+
+The `i3.8xlarge` instance type provides four NVMe instance storage disks. After applying your changes using `eksctl`, you can install the example EKS NVMe manifest:
+```bash
+$ kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/sig-storage-local-static-provisioner/refs/heads/master/helm/generated_examples/eks-nvme-ssd.yaml
+```
+
+This should create a StorageClass named `nvme-ssd`, and four PVs for each local NVMe instance storage disk attached to the instance:
```

Comment:
> `four PVs for each local NVMe instance`

Not super clear to me. So, this means we will have 4 PVs for each EC2 Instance? What if customer has workload across many nodes? Could you please clarify many node scenario in the doc?